src/configure.rs

```diff
@@ -190,7 +227,7 @@ impl File {
 
 pub fn apply_configuration(configuration: &Configuration) {
     // Decrypt the project's configuration files
-    decrypt_files_for_configuration(&configuration).expect("Unable to decrypt and copy files");
+    decrypt_files_for_configuration(configuration).expect("Unable to decrypt and copy files");
```

Comment:
This is required because `apply_configuration` takes a reference: https://github.com/Automattic/configure/blob/e76059cf27b55cbd6d57cdade15d63834d62ac36/src/configure.rs#L229

This could get reworked in the future, just wasn't been something that was _that_ important yet 😅

Reply:
Hmm.. maybe I didn't realize that the argument was already a reference when I first reviewed this 😕 It's been so long that I really can't remember anything, so... I'll just mark it as resolved. 😬